src/isFinite.js

```diff
@@ -0,0 +1,6 @@
+'use strict'
+
+const isFinite = (val) => (typeof val === 'number' && Number.isFinite(val)) ||
```

Comment:
I think this would be simplified like this:
```
const isFinite = (val) => isNumber(val) && Number.isFinite(val)
```

Reply:
hmm , in that case 
`isFinite(new Number(55)) = false ` 
because Number.isFinite(new Number(55)) is false

And it would be true

Reply:
fyi, this is from lodash:
```
   function isFinite(value) {
      return typeof value == 'number' && nativeIsFinite(value);
    }
```
so it is indeed simpler. Tbh, I really don't get the point of reinventing the wheel there, especially there is a pretty solid shinny wheel out there.

Reply:
Hi @bariscicek,
I totally agree with you with 'don't reinvent the wheel' because it is a 'solid shinny wheel', but here I'm having some doubts:
1.- If we don't want to "reinvent the wheel", why don't we fork the current version of lodash under bitfinexcom and use that?
2.- In the isString() and isNumber() methods, some teammates noted that 
`isString( new String('hello') ) ` returned false
`isNumber( new Number (55) ) ` returned false
IMO, we could avoid these cases, although Lodash managed these cases as true... 
but then lodash return false in this:
`_.isFinite(new Number(555))`  ---> false 
In IMO, it is an **error**, and I wanted to fix it in our lib, then what's the point here?
- keep or fix that 'error'? 
- if we don't consider that an error, why should we manage these cases in the isString and isNumber methods?
cc @vigan-abd

Reply:
If I may. I think is a little overkill to support that kind of cases. We opted for not doing that on tje kyc backend https://github.com/bitfinexcom/bfx-core-kyc-js/pull/1017 We used  a simple typeof for string, number and boolean.

Reply:
To close this topic, 
I changed the implementation to Vigan's suggestion.
`(val) => isNumber(val) && Number.isFinite(val)`

In this case, we are aware that 
`isFinite( new Number(24) ) => FALSE` 
we are not very consistent in accepting isNumber(new Number(4)) true and isFinite of that as False... but in any case Lodash did the same...

Reply:
tbh idk who uses new Number 😅, but indeed seems wrong isFinite being true check against number object, IMHO who uses new Number should use .valueOf to check against number being finite (isFinite(new Number(123).valueOf()))